### PR TITLE
[feature](profile) add MaxActiveTime and add exectime in pipelineX

### PR DESCRIPTION
**Pull request by Mryange**
## Proposed changes

before

```
Fragments: 
                                     ┌───────────────────────┐
                                     │[-1: VDataBufferSender]│
                                     │Fragment: 0            │
                                     │MaxActiveTime: 0ns     │
                                     └───────────────────────┘

```

now

```
Fragments: 
                                     ┌───────────────────────┐
                                     │[-1: VDataBufferSender]│
                                     │Fragment: 0            │
                                     │MaxActiveTime: 27s532ms│
                                     └───────────────────────┘    
```

now

```
SimpleProfile:
    Instance(1):
        PipelineXTask  (index=0):
        RESULT_SINK_OPERATOR  (id=0):
              -  CloseTime:  avg  10.569us,  max  10.569us,  min  10.569us
              -  ExecTime:  avg  285.258us,  max  285.258us,  min  285.258us
              -  InputRows:  0
              -  OpenTime:  avg  85.85us,  max  85.85us,  min  85.85us
        AGGREGATION_OPERATOR  (id=453):
              -  PlanInfo
                    -  output:  sum(partial_sum((lo_extendedprice  *  lo_discount)))[#45]
                    -  group  by:  
                    -  cardinality=1
              -  BlocksReturned:  1
              -  CloseTime:  avg  4.267us,  max  4.267us,  min  4.267us
              -  ExecTime:  avg  111.687us,  max  111.687us,  min  111.687us
              -  OpenTime:  avg  20.933us,  max  20.933us,  min  20.933us
              -  ProjectionTime:  avg  0ns,  max  0ns,  min  0ns
              -  RowsReturned:  1
            PipelineXTask  (index=1):
            AGGREGATION_SINK_OPERATOR  (id=453):
                  -  CloseTime:  avg  2.141us,  max  2.141us,  min  2.141us
                  -  ExecTime:  avg  160.937us,  max  160.937us,  min  160.937us
                  -  InputRows:  8
                  -  OpenTime:  avg  99.439us,  max  99.439us,  min  99.439us
            EXCHANGE_OPERATOR  (id=450):
                  -  PlanInfo
                        -  offset:  0
                  -  BlocksReturned:  8
                  -  CloseTime:  avg  13.803us,  max  13.803us,  min  13.803us
                  -  ExecTime:  avg  182.182us,  max  182.182us,  min  182.182us
                  -  OpenTime:  avg  91.845us,  max  91.845us,  min  91.845us
                  -  ProjectionTime:  avg  0ns,  max  0ns,  min  0ns
                  -  RowsReturned:  8
```

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

File: be/src/util/runtime_profile.h [hunk 1]
```diff
@@ -333,7 +333,12 @@ class RuntimeProfile {
     const std::string* get_info_string(const std::string& key);
 
     // Returns the counter for the total elapsed time.
-    Counter* total_time_counter() { return &_counter_total_time; }
+    Counter* total_time_counter() {
```
Comment (yiguolei) [inline be/src/util/runtime_profile.h:336]: 不要这么改， 直接加一个新方法   exec_time_counter()， 你这么改，当我们看pipelinex的代码的时候，看profile 会很疑惑。

File: be/src/util/runtime_profile.h [hunk 2]
```diff
@@ -428,6 +433,8 @@ class RuntimeProfile {
 
     void clear_children();
 
+    void set_pipelineX_exec_time(Counter* exec_timer) { _pipelineX_exec_time = exec_timer; }
```
Comment (yiguolei) [inline be/src/util/runtime_profile.h:436]: 尽量不要写这种set
这种写法会导致指针的生命周期很难控制

**Comment by Mryange** [comment]
run buildall

**Review by yiguolei — DISMISSED** [review]
LGTM

**Comment by Mryange** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]
LGTM

**Review by Gabriel39 — APPROVED** [review]

**Pull request merged**